python/vyos/ifconfig/interface.py

```diff
@@ -795,6 +872,10 @@ def add_ipv6_eui64_address(self, prefix):
         Calculate the EUI64 from the interface's MAC, then assign it
         with the given prefix to the interface.
         """
+        # Don't allow in netns yet
```

Comment:
This just looks like adding a normal address. Does the mac address change when the interface is moved/recreated? It feels to me like this should already work and should be allowed